src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -1,55 +1,172 @@
 import React from 'react';
-import {userEvent} from '@testing-library/user-event';
-import {render, screen} from '../../../test-utils/testing-library';
-import BoxAISidebarComponent, {BoxAISidebarProps} from '../BoxAISidebar';
+import { userEvent } from '@testing-library/user-event';
+import { screen, render } from '../../../test-utils/testing-library';
+import BoxAISidebar, { BoxAISidebarProps } from '../BoxAISidebar';
 
-const mockOnClearClick = jest.fn();
+jest.mock('@box/box-ai-agent-selector', () => ({
+    ...jest.requireActual('@box/box-ai-agent-selector'),
+    BoxAiAgentSelectorWithApi: () => <div data-testid="sidebar-agent-selector" />
+}));
+
+const mockOnClearAction = jest.fn();
```

Comment:
can we move this into the test that actually needs this? the rest of the test dont need this spy

Reply:
Not without changing how we pass values to the wrapper, and this I'd like to avoid as mentioned above